migrations/20210421143121-remove-PlatformTipForTransactionGroup.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    // Migrate all existing platform tips to make sure their TransactionGroup are the same
+    // as the transactions they're referring to.
+    await queryInterface.sequelize.query(`
+      UPDATE "Transactions"
+      SET "TransactionGroup" = "PlatformTipForTransactionGroup"::uuid
+      WHERE "PlatformTipForTransactionGroup" IS NOT NULL
+      AND "PlatformTipForTransactionGroup"::uuid != "TransactionGroup"
+      AND (data ->> 'isFeesOnTop')::boolean = TRUE
+    `);
+
+    // Remove `PlatformTipForTransactionGroup`
+    await queryInterface.removeColumn('Transactions', 'PlatformTipForTransactionGroup');
+
+    // Add an index on `TransactionGroup`, since we're going to use it more intensively from now on
+    await queryInterface.addIndex('Transactions', ['TransactionGroup'], { concurrently: true });
```

Comment:
This one might take some time. Do you expect it to be blocking or not?

Reply:
I recently discovered the magic of `concurrently: true` when creating indexes. It's my first time using it, but it normally guarantees that we won't block the DB :raised_hands: